SkycoinProject/skywire-mainnet
# Feature/dmsg hypervisor

Fixes 
https://github.com/skycoin/skywire/issues/560

 Changes:	
- Now communication visor-hypervisor is thru dmsg

How to test this PR:
- Within skywire-services under `feature/dmsg-hypervisor ` run `integration-run-generic` from skywire-services and check that visors have been able to contact the hypervisor. You can check it by performing any query that returns visor-provided information to the hypervisor, like `curl localhost:8080/api/nodes`.
